ln-simln-jamming/src/main.rs

```diff
@@ -156,14 +134,25 @@ async fn main() -> Result<(), BoxError> {
         }
     });
 
-    let reputation_interceptor = Arc::new(Mutex::new(ReputationInterceptor::new_for_network(
-        forward_params,
-        &sim_network,
-        reputation_check,
-        clock.clone(),
-        Some(results_writer),
-        shutdown.clone(),
-    )?));
+    let reputation_dir = &cli.reputation_dir.join(cli.attacker_bootstrap.0.clone());
```

Comment:
Not for this PR, but we've got a slight inconsistency between write and read of these snapshots. 
`attacker_bootstrap` is optional for write but required for read, so we can generate a file that has no attacker bootstrap but can't run with it (becuase we require the CI flag). 

I think we can rela that requirement in a follow up?